src/libAtomVM/context.h

```diff
@@ -174,4 +174,19 @@ static inline int context_is_waiting_timeout(const Context *ctx)
     return ctx->timeout_at.tv_sec || ctx->timeout_at.tv_nsec;
 }
 
+/**
+ * @brief Returns a term representing an atom, from the suppliend string
+ *
+ * @details Converns a string to an atom.  Note that this function may have a side-effect on the
+ *          global context.
+ * @param glb pointer to the global context
+ * @param string an AtomString
+ * @return an atom term formed from the supplied atom string.
+ */
+static inline term context_from_atom_string(Context *ctx, AtomString string)
```

Comment:
It might be misleading, `context_from_atom_string` sounds like a Context is created from an AtomString. Is make_atom better?

Reply:
I thought that the coding conventions required the function names to be prefixed with their module names?

https://github.com/bettio/AtomVM/blame/master/CONTRIBUTING.md#L50